index.html

```diff
@@ -0,0 +1,102 @@
+                    </a>
+                    <div class="search-bar">
+                        <input class="nav-input-container fa" placeholder="&#xF002 Search"></input>
+                    </div>
+                    <div class="nav-action-buttons">
+                        <div class="dropdown">
+                            <a href="/Signup/Signup.html"><i class="fas fa-user fa-lg"></i></a>
+                            <div class="dropdown-content">
+                                <div class="dropdown-items">
+                                    <a href="#">Orders</a>
+                                    <a href="#">Wishlist</a>
+                                </div>
+
+                            </div>
+                        </div>
+                        <div class="icon-ecomm">
+                            <a href="/WishList/Wishlist.html"><i class="far fa-heart fa-lg"></i></a>
+                        </div>
+                        <div class="icon-ecomm">
+                            <a href="/Cart/Cart.html"><i class="fa fa-shopping-cart" aria-hidden="true"></i>
+                            </a>
+                        </div>
+                </nav>
+            </div>
+        </header>
+
+        <div class="main-teaser">
+            <div class="hero-container">
+                <a href="/Product/Product.html" class="btn-primary">
+                    <img class="hero-image" src="./assets/images/hero-teaser.jpg" alt="" />
+                </a>
+            </div>
+        </div>
+
+        <div class="category-container">
+            <h2 class="category-headline">Products</h2>
+
+            <div class="category-list">
+
+                <div class="img img-lg">
+                    <a href="/Product/Products.html" class="btn-primary">
+                        <img src="/assets/images/home_categoreis/apparel.jpg" />
+                        <h2 class="category-title">Apparel</h2>
+                    </a>
+                </div>
+
+                <div class="img img-lg">
+                    <a href="/Product/Products.html" class="btn-primary">
+                        <img src="/assets/images/home_categoreis/babycare.jpg" />
+                        <h2 class="category-title">Baby Care</h2>
+                    </a>
+                </div>
+
+
+                <div class="img img-lg">
+                    <a href="/Product/Products.html" class="btn-primary">
+                        <img src="/assets/images/home_categoreis/toys.jpg" />
+                        <h2 class="category-title">Toys</h2>
+                    </a>
+                </div>
+
+
+                <div class="img img-lg">
+                    <a href="/Product/Products.html" class="btn-primary">
+                        <img src="/assets/images/home_categoreis/travel.jpg" />
+                        <h2 class="category-title">Travel</h2>
+                    </a>
+                </div>
+
+
+            </div>
+        </div>
+
```

Comment:
clean code ! everything looks good ! GTM

Reply:
Thank you